tests/scripts/thread-cert/Cert_5_2_06_RouterDowngrade.py

```diff
@@ -33,175 +33,221 @@
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         13: {
+            'name': 'ROUTER_12',
             'mode': 'rdn',
             'panid': 0xface,
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         14: {
+            'name': 'ROUTER_13',
             'mode': 'rdn',
             'panid': 0xface,
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         15: {
+            'name': 'ROUTER_14',
             'mode': 'rdn',
             'panid': 0xface,
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         16: {
+            'name': 'ROUTER_15',
             'mode': 'rdn',
             'panid': 0xface,
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         17: {
+            'name': 'ROUTER_16',
             'mode': 'rdn',
             'panid': 0xface,
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         18: {
+            'name': 'ROUTER_17',
             'mode': 'rdn',
             'panid': 0xface,
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         19: {
+            'name': 'ROUTER_18',
             'mode': 'rdn',
             'panid': 0xface,
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         20: {
+            'name': 'ROUTER_19',
             'mode': 'rdn',
             'panid': 0xface,
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         21: {
+            'name': 'ROUTER_20',
             'mode': 'rdn',
             'panid': 0xface,
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         22: {
+            'name': 'ROUTER_21',
             'mode': 'rdn',
             'panid': 0xface,
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         23: {
+            'name': 'ROUTER_22',
             'mode': 'rdn',
             'panid': 0xface,
             'router_downgrade_threshold': 32,
             'router_selection_jitter': 1,
             'router_upgrade_threshold': 32
         },
         ROUTER24: {
+            'name': 'ROUTER_23',
```

Comment:
it's a little confusing why ROUTER24 is `ROUTER_23`. Maybe change `ROUTER24` to `ROUTER23` ?

Reply:
changed to ROUTER23